packages/shared/src/dto/step-schemas/generate-preview-response.dto.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable */
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Same as above, please specify a specific eslint rule.‏